apps/src/studio/studio.js

```diff
@@ -1142,11 +1144,13 @@ Studio.clearEventHandlersKillTickLoop = function() {
     });
   }
   Studio.eventHandlers = [];
-  if (Studio.intervalId) {
-    window.clearInterval(Studio.intervalId);
-  }
+  if (Studio.perExecutionTimeouts) {
+    Studio.perExecutionTimeouts.forEach(function (timeout) {
+      clearInterval(timeout);
+    });
+   }
```

Comment:
I'm not so familiar with that code, but I see that it's modeled on the eventHandlers code immediately above it which works similarly.  Studio.clearEventHandlersKillTickLoop is the first thing called by Studio.Init which is why they each follow this pattern I guess.